inc/session.class.php

```diff
@@ -1082,22 +1082,20 @@ static function addMessageAfterRedirect($msg, $check_once = false, $message_type
                 || !isset($array[$message_type])
                 || in_array($msg, $array[$message_type]) === false) {
                array_push($array[$message_type], $msg);
-               global $IS_TWIG, $container;
-               if ($IS_TWIG) {
-                  switch ($message_type) {
-                     case ERROR:
-                        $flashtype = 'error';
-                        break;
-                     case WARNING:
-                        $flashtype = 'warniing';
-                        break;
-                     case INFO:
-                     default:
-                        $flashtype = 'info';
-                        break;
-                  }
-                  $container->get(Slim\Flash\Messages::class)->addMessage($flashtype, $msg);
+               global $CONTAINER;
+               switch ($message_type) {
+                  case ERROR:
+                     $flashtype = 'error';
+                     break;
+                  case WARNING:
+                     $flashtype = 'warniing';
```

Comment:
I think 'warniing' is a typo, it should be 'warning'

Reply:
Good catch, thanks!